banking_system.rb

```diff
@@ -0,0 +1,133 @@
+class Account
+	@@number = 23456789101
+	@@balance = 0
+	def initialize
+		@acc_number = @@number
+	end
+
+	def open_account
+		@@number += 1
+		@acc_number = @@number
+	end
+
+	def deposit_money(amount)
+		@@balance += amount
+	end
+
+	def withdraw_money(amount)
+		if @@balance <= 0 || @@balance <= amount
+			return false
+		else
+			@@balance -= amount
+			return true
+		end
+	end
+
+	def get_balance
+		return @@balance
+	end
+
+	def interest(rate,time)
+		interst = (@@balance*rate*time)/100
+	end
```

Comment:
Give proper spaces. In ruby we highly value readability.
Change it to
```ruby
def interest(rate, time)
  (@@balance * rate * time) / 100
end
```

Reply:
Also, this logic will calculate the wrong interest, as the balance is shared with all persons it will show give the same output for regardless of an individual's balance.